public/js/fetch.js

```diff
@@ -1,14 +1,14 @@
-const fetch = (method, url, callback) => {
+const fetch = (value, method, url, callback) => {
   const xhr = new XMLHttpRequest();
   xhr.onreadystatechange = () => {
     if (xhr.readyState === 4)
-      if (xhr.status !== 200) alert("Error!!!!");
+      if (xhr.status !== 200) callback(null, xhr.status);
```

Comment:
why not making it this way:
const error = `Error, ${xhr.status}`
callback(error);